tests/jvm/src/test/scala/scala/meta/tests/semanticdb/TargetedSuite.scala

```diff
@@ -1315,4 +1316,20 @@ class TargetedSuite extends DatabaseSuite(SemanticdbMode.Slim) {
       }
     }
   )
+
+  targeted(
+    """
+      |object a {
+      |  for {
+      |    i <- List(1, 2)
+      |    <<j>> <- List(3, 4)
+      |  } yield j
+      |}
+      """.stripMargin, { (db, j) =>
+      val denot = db.symbols.find(_.symbol == j).get.denotation
+      val kind = denot.skind
+
+      assert(kind == k.LOCAL)
```

Comment:
For consistency with https://github.com/scalameta/scalameta/pull/1501#discussion_r181561316, maybe make this `k.isLocal`?

Reply:
That's a good point, I agree. Gonna merge nevertheless so @andyscott can get a t-shirt 😂

Reply:
I won't stand between a man and his t-shirt 😂

Reply:
👕👍